research/src/pages/components/selectmenu.mdx

```diff
@@ -0,0 +1,257 @@
+---
+name: selectmenu explainer
+path: /components/selectmenu
```

Comment:
i don't think this is necessary since we automatically build the path using the file directory

Reply:
done